src/studio/nuxeo-studio/working-in-studio/migration-services.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration Services
+review:
+    comment: ''
+    date: '2018-12-05'
+    status: ok
+tree_item_index: 950
+toc: true
+---
+The JSF UI to Web UI migration tool is a safe and automated way to create, within a few minutes, an equivalent Web UI configuration based on your existing JSF UI configuration.
+
+The migration tool reduces dramatically the effort needed to transition to [Web UI]({{page space="nxdoc" page="web-ui"}}), our user interface that uses modern web standard technologies to provide an outstanding user experience.
+
+Running the migration tool does not delete any data. If you are not satisfied with the result, you can:
+* go back to the state your project was before triggering it anytime,
+* delete the additional configuration that was generated partially or totally.
+
+After migration is run, a detailed report is created in **Migration Services** so that you can check the results.
+
+{{#> callout type='info' heading='part of early access program'}}
+JSF UI to Web UI migration tool is available through our early access program, offering stable features you can co-create with us as they are only partially complete.<br /><br />
+
+- Not a member of it yet? [Learn about its benefits, what's planned and how to get in]({{page page="early-access-program"}}).
+<br /><br />
+- Already a member? Help us co-creating it by [telling us about your experience](https://docs.google.com/forms/d/e/1FAIpQLSdoDfpJOnFwkxHAHOk5aq5KHh9myxF6fISugnMzVmvuQSp-fw/viewform?usp=pp_url) </a>.
+{{/callout}}
+
+## What is migrated?
```

Comment:
On this section, maybe I explain how each items is technically converted

Reply:
Is this in the scope of ticket https://jira.nuxeo.com/browse/NXDOC-1551 ?